CHANGELOG.md

```diff
@@ -0,0 +1,639 @@
+- Add cw20 support to token-weighted group [\#143](https://github.com/CosmWasm/cw-plus/issues/143)
+
+**Merged pull requests:**
+
+- Cosmwasm beta5 [\#287](https://github.com/CosmWasm/cw-plus/pull/287) ([ethanfrey](https://github.com/ethanfrey))
+- Cw20ReceiveMsg msg field [\#286](https://github.com/CosmWasm/cw-plus/pull/286) ([maurolacy](https://github.com/maurolacy))
+- Fix ci contract build [\#285](https://github.com/CosmWasm/cw-plus/pull/285) ([ethanfrey](https://github.com/ethanfrey))
+- Use Cw20 token in cw4-stake [\#282](https://github.com/CosmWasm/cw-plus/pull/282) ([ethanfrey](https://github.com/ethanfrey))
+- Avoid the need for Any type by using Empty as message type and String as error type [\#281](https://github.com/CosmWasm/cw-plus/pull/281) ([webmaster128](https://github.com/webmaster128))
+- Update to 0.14.0 beta4 [\#280](https://github.com/CosmWasm/cw-plus/pull/280) ([maurolacy](https://github.com/maurolacy))
+- Better error message with missing sudo \(no parse error\) [\#279](https://github.com/CosmWasm/cw-plus/pull/279) ([ethanfrey](https://github.com/ethanfrey))
+
+## [v0.6.0-beta2](https://github.com/CosmWasm/cw-plus/tree/v0.6.0-beta2) (2021-04-19)
+
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.6.0-beta1...v0.6.0-beta2)
+
+**Closed issues:**
+
+- Add secondary index support to SnapshotMap. [\#255](https://github.com/CosmWasm/cw-plus/issues/255)
+
+**Merged pull requests:**
+
+- Indexed snapshot. Expose primary methods [\#275](https://github.com/CosmWasm/cw-plus/pull/275) ([maurolacy](https://github.com/maurolacy))
+- Indexed snapshot map [\#271](https://github.com/CosmWasm/cw-plus/pull/271) ([maurolacy](https://github.com/maurolacy))
+- Run clippy on test code [\#270](https://github.com/CosmWasm/cw-plus/pull/270) ([webmaster128](https://github.com/webmaster128))
+
+## [v0.6.0-beta1](https://github.com/CosmWasm/cw-plus/tree/v0.6.0-beta1) (2021-04-13)
+
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.6.0-alpha3...v0.6.0-beta1)
+
+**Closed issues:**
+
+- Update to cosmwasm v0.14.0-beta3 [\#268](https://github.com/CosmWasm/cw-plus/issues/268)
+- My [\#263](https://github.com/CosmWasm/cw-plus/issues/263)
+
+**Merged pull requests:**
+
+- Bump dependency to cosmasm v0.14.0-beta3 [\#269](https://github.com/CosmWasm/cw-plus/pull/269) ([ethanfrey](https://github.com/ethanfrey))
+- Remove unused PrimaryKey::parse\_key [\#267](https://github.com/CosmWasm/cw-plus/pull/267) ([webmaster128](https://github.com/webmaster128))
+- Use workspace-optimizer:0.11.0 [\#262](https://github.com/CosmWasm/cw-plus/pull/262) ([webmaster128](https://github.com/webmaster128))
+- Update cosmwasm-std [\#260](https://github.com/CosmWasm/cw-plus/pull/260) ([yihuang](https://github.com/yihuang))
+- implement demo cw1155 contract [\#251](https://github.com/CosmWasm/cw-plus/pull/251) ([yihuang](https://github.com/yihuang))
+
+## [v0.6.0-alpha3](https://github.com/CosmWasm/cw-plus/tree/v0.6.0-alpha3) (2021-04-01)
```

Comment:
Maybe you can manually merge these -alpha releases together as well?

Reply:
Done.